src/base/static/components/form-fields/types/datetime-field.js

```diff
@@ -2,16 +2,24 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import Datetime from "react-datetime";
 import moment from "moment";
+import classNames from "classnames";
 
 import "react-datetime/css/react-datetime.css";
 
 import constants from "../../../constants";
 
 import "./datetime-field.scss";
 
+// HACK: We use a far-future timestamp to represent the notion of an "ongoing"
+// datetime selection.
+// TODO: Support the ability for field components to write to multiple backend
+// data fields of different data types.
+const ONGOING_DATETIME = "9999-12-31 23:59:59";
+
 class DatetimeField extends Component {
   componentDidMount() {
     this.props.isAutoFocusing && this.inputRef.focus();
```

Comment:
Just curious - is `props.isAutFocusing` ever `false` here?

Reply:
Yes, in fact it's almost always `false`. It's only `true` when it's triggered by another component (as is the case on the snohomish input form, where an action checkbox triggers and then focuses on its corresponding datetime field).

Reply:
That makes sense - thanks for explaining!